### PR TITLE
Support for pi2 clock values

### PR DESCRIPTION
**Pull request by Ealdwulf**
See commit messages for more explanation. Fixes #1642
Note that there are many cases that this has to support.

### CODE REVIEW

**Comment by Ealdwulf** [comment]
@alex5imon It looks like you wrote overclock.py, do you want to review this?

**Comment by alex5imon** [comment]
We might want to rename set-kano-hdmi file, since we are not just doing hdmi related stuff in there

File: kano_settings/boot_config.py [hunk 1]
```diff
@@ -5,98 +5,135 @@
 # Copyright (C) 2014 Kano Computing Ltd.
```
Comment (alex5imon) [inline kano_settings/boot_config.py]: addd `, 2015`

File: kano_settings/system/overclock_chip_support.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# set-kano-hdmi
```
Comment (alex5imon) [inline kano_settings/system/overclock_chip_support.py]: change this name

**Comment by alex5imon** [comment]
@Ealdwulf there are only a few minor comments from my side. Feel free to merge when you apply the changes

**Comment by Ealdwulf** [comment]
Thanks! Will do.

**Comment by alex5imon** [comment]
Please merge when ready :+1:

**Comment by Ealdwulf** [comment]
NB I also renamed /etc/init.d/kano-hdmi to /etc/init.d/kano-settings. 
It turns out dpkg won't remove the old file kano-hdmi, but I think it is safe because the link to it in /etc/rc2.d is removed.

**Pull request merged**